man/tpm2_sign.1.md

```diff
@@ -46,6 +46,14 @@ data and validation shall indicate that hashed data did not start with
 
     The message file, containing the content to be  digested.
 
+  * **-D**, **--digest**=_DIGEST\_FILE_:
+
+    The digest file that shall be computed using the correct hash
+    algorithm. When this option is specified, both the message file and
+    the validation ticket are ignored.
+    You cannot use this option to sign a digest against a restricted
+    signed key.
```

Comment:
"signing key" right, not "signed key" here?

Reply:
Yes, sorry :)